server/src/game/map/tile/city.ts

```diff
@@ -67,6 +68,15 @@ export class City {
     return this.tiles;
   }
 
+  /**
+   * 
+   * @returns the "controlling" domainID: if this a city state, return the city's own ID. Else, return the parent Civ's ID.
+   */
+  getControllingDomainID(): DomainID {
+    if (!this.civID) return this.getDomainID();
+    else return this.civID;
```

Comment:
Small thing, but personally I would write this as `return this.civID ?? this.getDomainID()`. I suppose that's a matter of taste, though.

Reply:
I agree, actually.